analytics_data_api/v0/documents.py

```diff
@@ -160,7 +160,10 @@ def get_users_in_course(
             }
             for sort_policy in sort_policies
         ])
-        return search_request.execute()
+
+        # by default, ES will only return the first 10 documents, we want to get as many as possible
+        total_search_requests = search_request[0:10000]
```

Comment:
I hate to add stuff now but I think we should do the count solution and do
count
log about the clamp if count > 10000 
do search request [0:count or 10000]

so we can see how much trouble is happening

Reply:
We can do this on monday

Reply:
fair, we should log this but you could add that later if you want